projects/plugins/boost/wp-js-data-sync.php

```diff
@@ -162,5 +166,8 @@ function jetpack_boost_initialize_datasync() {
 $css_excludes_entry = new Minify_Excludes_State_Entry( 'minify_css_excludes' );
 jetpack_boost_register_option( 'minify_js_excludes', Schema::as_array( Schema::as_string() )->fallback( Minify_JS::$default_excludes ), $js_excludes_entry );
 jetpack_boost_register_option( 'minify_css_excludes', Schema::as_array( Schema::as_string() )->fallback( Minify_CSS::$default_excludes ), $css_excludes_entry );
+jetpack_boost_register_option( 'image_cdn_quality', ( new Schema( new Type_Constrained_Number( 1, 100 ) ) )->fallback( 100 ) );
```

Comment:
I think the fallback should be whatever the default value for photon quality slider is right now